crates/iceberg/src/expr/visitors/inclusive_metrics_evaluator.rs

```diff
@@ -2104,6 +2113,9 @@ mod test {
             equality_ids: vec![],
             sort_order_id: None,
             partition_spec_id: 0,
+            referenced_data_file: None,
```

Comment:
The reason I manually assigned these fields is to keep coding style consistency.